src/components/web3/StartCall.tsx

```diff
@@ -25,12 +27,15 @@ export const StartCall: React.FC<Props> = ({
   setJitsiContext,
   isSubscribed,
 }) => {
+  console.log(useParams());
```

Comment:
```suggestion
```

either remove this log or add some extra text so people reading the console understand what is being logged and why

Reply:
@davidsuh11 - i agree with @tackley, we really don't need this particular call to `console.log` ... if you agree, please remove it and i'll approve and we'll get it into `main` ... thanks!